api/package.json

```diff
@@ -15,7 +15,10 @@
     "babel-polyfill": "^6.26.0",
     "dotenv": "^5.0.1",
     "elasticsearch": "^14.0.0",
-    "express-inspector": "^0.3.1"
+    "express-inspector": "^0.3.1",
+    "fs": "^0.0.1-security",
+    "googleapis": "27",
+    "mongoose": "^5.1.3"
```

Comment:
let's not mix the search api with CMS api..could we refactor these things to another folder called "cms" or "cms-api" or anything else that seems more suitable? so the repo will have 4 root folders:
/api
/cms
/data_model
/ui